src/silo_api/request_handler.test.cpp

```diff
@@ -0,0 +1,74 @@
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "silo_api/manual_poco_mocks.test.h"
+#include "silo_api/request_handler.h"
+
+class MockRequestHandler : public Poco::Net::HTTPRequestHandler {
+  public:
+   MOCK_METHOD(
+      void,
+      handleRequest,
+      (Poco::Net::HTTPServerRequest & request, Poco::Net::HTTPServerResponse& response),
+      ()
+   );
+};
+
+TEST(ErrorRequestHandler, handlesRuntimeErrors) {
```

Comment:
In other test cases we used snake case. Which do we prefer? Google says, that CamelCase would be better, because of strange leading underscore stuff. We should add this to the README

Reply:
The docs say that one should not use `_` in test names: https://github.com/google/googletest/blob/main/docs/faq.md#why-should-test-suite-names-and-test-names-not-contain-underscore

Reply:
I'll have a look how to disable the clang-tidy functionality for this. Then I will change the test names accordingly. Maybe in a separate pull request.